classes/Link.php

```diff
@@ -882,7 +882,7 @@ public function getTabLink(array $tab)
 
     /**
      * Used when you explicitly want to create a LEGACY admin link, this should be deprecated
-     * in 1.8.0.
+     * in 9.x
```

Comment:
whats the point of having this whole comment if by every version we just move it further to be deprecated in next version?
Either it is deprecated or not. (no "should be deprecated someday")

Reply:
Let's deprecate it in the next PR 👍🏻